tests/api/test_referencefile.py

```diff
@@ -0,0 +1,101 @@
+from collections import namedtuple
+
+import pytest
+from starlette.testclient import TestClient
+
+from agr_literature_service.api.main import app
+from fastapi import status
+
+from agr_literature_service.api.models import ReferencefileModel
+from agr_literature_service.lit_processing.tests.mod_populate_load import populate_test_mods
+from .test_reference import test_reference # noqa
+from ..fixtures import db # noqa
+from .fixtures import auth_headers # noqa
+
+
+TestReferencefileData = namedtuple('TestReferencefileData', ['response', 'new_referencefile_id'])
+
+
+@pytest.fixture
+def test_referencefile(db, auth_headers, test_reference): # noqa
+    print("***** Adding a test referencefile *****")
+    with TestClient(app) as client:
+        new_referencefile = {
+            "display_name": "Bob",
+            "reference_curie": test_reference.new_ref_curie,
+            "file_class": "main",
+            "file_publication_status": "final",
+            "file_extension": "pdf",
+            "pdf_type": "pdf",
+            "md5sum": "1234567890"
```

Comment:
Might want another test with is_annotation true/false, since it's a different type of variable, and maybe another test with a null pdf_type to check that also works

Reply:
added the suggested tests, thanks @azurebrd